<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="box" native="true">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>291</width>
      <height>111</height>
     </rect>
    </property>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter subject name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="period">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter number of periods:</string>
     </property>
    </widget>
    <widget class="QLabel" name="teach">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter name of teachers:
(separated by commas)</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ename">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="eperiod">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="eteach">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Algerian</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Enter Subject</string>
    </property>
   </widget>
   <widget class="QPushButton" name="submit1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Submit</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>347</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>submit1</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>221</x>
     <y>-5</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submit1</sender>
   <signal>pressed()</signal>
   <receiver>submit1</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>205</x>
     <y>198</y>
    </hint>
    <hint type="destinationlabel">
     <x>203</x>
     <y>214</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submit1</sender>
   <signal>pressed()</signal>
   <receiver>submit1</receiver>
   <slot>showMaximized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>129</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>118</x>
     <y>205</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
